src/targets/happymodel_2400.ini

```diff
@@ -0,0 +1,77 @@
+
+# ********************************
+# Transmitter targets
+# ********************************
+
+[env:HappyModel_ES24TX_2400_TX_via_UART]
+extends = env_common_esp32
+board_build.f_cpu = 160000000L
+build_flags =
+	${env_common_esp32.build_flags}
+	${common_env_data.build_flags_tx}
+	-D TARGET_TX_ESP32_E28_SX1280_V1
+	-D TARGET_SX1280
+	-D VTABLES_IN_FLASH=1
+	-O2
+src_filter = ${env_common_esp32.src_filter} -<rx_*.cpp>
+
+[env:HappyModel_ES24TX_2400_TX_via_WIFI]
+extends = env:HappyModel_ES24TX_2400_TX_via_UART
+
+
+# ********************************
+# Receiver targets
+# ********************************
+
+[env:HappyModel_EP_2400_RX_via_UART]
+extends = env_common_esp82xx
+build_flags =
+	${env_common_esp82xx.build_flags}
+	${common_env_data.build_flags_rx}
+	-D TARGET_RX_ESP8266_SX1280_V1
+	-D TARGET_100mW_MODULE
+	-D TARGET_SX1280
+src_filter = ${env_common_esp82xx.src_filter} -<tx_*.cpp>
+
+[env:HappyModel_EP_2400_RX_via_BetaflightPassthrough]
+extends = env:HappyModel_EP_2400_RX_via_UART
+upload_protocol = custom
+upload_command =
+	python $PROJECT_DATA_DIR/../python/BFinitPassthrough.py $UPLOAD_SPEED
+	python $PROJECT_DATA_DIR/../python/esptool-3.0/esptool.py --no-stub -b $UPLOAD_SPEED -c esp8266 --before no_reset --after soft_reset write_flash 0x0000 $SOURCE
+
+[env:HappyModel_EP_2400_RX_via_WIFI]
+extends = env:DIY_2400_RX_ESP8285_SX1280_via_UART
+upload_port = 10.0.0.1
+
+[env:HappyModel_PP_2400_RX_via_STLINK]
+extends = env_common_stm32
```

Comment:
Dont copy whole block here. Use `extends` instead

Reply:
Same applies for others as well. These are basically just renamed targets.

Reply:
Updated.